sqlx-core/src/migrate/source.rs

```diff
@@ -97,7 +97,7 @@ pub fn resolve_blocking(path: PathBuf) -> Result<Vec<(Migration, PathBuf)>, Reso
         let parts = file_name.splitn(2, '_').collect::<Vec<_>>();
 
         if parts.len() != 2 || !parts[1].ends_with(".sql") {
-            // not of the format: <VERSION>_<DESCRIPTION>.sql; ignore
+            // not of the format: <VERSION>_<DESCRIPTION>.<REVERSIBLE_DIRECTION>.sql; ignore
```

Comment:
Drive-by-comment here, but if the "tag" for no transaction is now inside the file itself, this comment is not right anymore?

Reply:
@kujeger I removed the `no_tx` piece from the comment but left the direction, is that what you are referring to?